graph-db/serverless.yml

```diff
@@ -13,33 +13,31 @@ provider:
    name: graphdb-driver-deployment-bucket-${self:provider.stage}
    skipPolicySetup: true
 
-  vpc:
-    securityGroupIds:
-      - ${cf:${self:provider.stage}.servicesSecurityGroupId}
-    subnetIds: !Split [',', '${cf:${self:provider.stage}.subnetIds}']
+  # vpc:
```

Comment:
Is this from local testing or not needed now?

Reply:
We need to pull the management of the infrastructure into this repository before we can do that. I've added a note for now that it needs to be commented locally.